1-getting-started.ipynb

```diff
@@ -4,7 +4,7 @@
    "cell_type": "markdown",
```

Comment:
_Commented on notebook `1-getting-started.ipynb` Cell 19 Line 5_
```markdown
### Quitting a command

Sometimes you need to terminate a command. This is often possible with the ctrl+c key-board command. 

In documentation you might see this represented as ^ C, where ^ is a symbol indicating the ‘ctrl‘ key.
```
In gitbook/markdown you can use `<kbd>CTRL</kbd>` to markup keyboard keys!